[workspace] Harden pnpm-workspace pattern against ad-hoc apps/* directories

## Context

\`pnpm-workspace.yaml\` currently uses a glob with explicit exclusions:

\`\`\`yaml
packages:
  - "packages/*"
  - "apps/*"
  - "!apps/wittgenstein-kimi"
  - "!apps/_vercel-kimi-out"   # added by #462 after this trip
\`\`\`

Every time a new ad-hoc subdirectory lands in \`apps/\` (locally generated build output, vendored upstream, scratch deck), the glob includes it. If that directory contains a \`package.json\`, \`pnpm install --frozen-lockfile\` trips:

\`\`\`
ERR_PNPM_OUTDATED_LOCKFILE  ...
specifiers in the lockfile ({}) don't match specs in package.json ({"sharp":"^0.33.5"})
\`\`\`

[#462](https://github.com/p-to-q/wittgenstein/pull/462) added the immediate fix for \`_vercel-kimi-out\`. This issue tracks the broader defensive move.

## Proposed change

Switch from glob + exclusions to an explicit allowlist:

\`\`\`yaml
packages:
  - "packages/*"          # all real codec / harness packages
  - "apps/site"           # canonical website
  - "apps/presentation"   # subject to #<presentation triage issue>
\`\`\`

\`packages/*\` remains glob because every entry is intentional. \`apps/*\` is rare enough (3-4 entries) that the allowlist is light maintenance and removes a class of foot-gun.

## Acceptance

- \`pnpm-workspace.yaml\` uses explicit allowlist for \`apps/*\` entries.
- A locally-generated \`apps/<anything-else>/package.json\` does not break \`pnpm install --frozen-lockfile\`.
- The doctor / smoke / publish-surface guards still pass.

## Non-goals

- Do not change \`packages/*\` to allowlist. Keep the glob — that surface is intentional.
- Do not relitigate the \`apps/wittgenstein-kimi\` exclusion design (it has its own \`pnpm-lock.yaml\` per Issue #112).

## Surfaced by

The 2026-05-26 maintainer cross-section audit ([#462](https://github.com/p-to-q/wittgenstein/pull/462)) when local Kimi deck build output broke the frozen install.

--- a/.github/workflows/status-labeler.yml
+++ b/.github/workflows/status-labeler.yml
@@ -141,7 +141,7 @@ jobs:
 
             function shouldEscalateNeedsMlSpecialistFromText(pr) {
               const text = `${pr.title || ""}\n${pr.body || ""}`;
-              return /Visual Seed Code|\\bVSC\\b|SeedExpander|decoder bridge/i.test(text);
+              return /Visual Seed Code|\bVSC\b|SeedExpander|decoder bridge/i.test(text);
             }
 
             async function addNeedsMlSpecialistIfApplicable(pr) {

--- a/apps/site/src/sections/HumanReadabilitySection.tsx
+++ b/apps/site/src/sections/HumanReadabilitySection.tsx
@@ -50,7 +50,7 @@ export default function HumanReadabilitySection() {
           >
             <span className="text-[hsl(var(--coral))]">User prompt</span>
             {"\n    →  schema preamble + "}
-            <span className="text-[hsl(var(--coral))]">Visual Seed Code (VSC)</span> - bearing image contract
+            <span className="text-[hsl(var(--coral))]">Visual Seed Code (VSC)</span>-bearing image contract
             {"\n          →  "}
             <span className="text-[hsl(var(--coral))]">LLM</span>
             {" (planner)"}

--- a/docs/research/2026-05-26-m1b-pr-boundary-audit.md
+++ b/docs/research/2026-05-26-m1b-pr-boundary-audit.md
@@ -98,7 +98,7 @@ split out into a separate governance/docs hygiene PR, or left out of the M1B PR:
 - `docs/v02-alignment-review.md`
 - `docs/v02-final-audit.md`
 
-Observed pattern: these changes replace `max` / `max.zhuang.yan@gmail.com` with
+Observed pattern: these changes replace a contributor's personal name / email with
 `@Jah-yee @Moapacha`. That is governance / attribution hygiene, not decoder
 delivery. Mixing it into M1B would expand review scope and trigger
 doctrine-surface review obligations unrelated to #334/#335/#402.

--- a/packages/cli/src/commands/doctor.ts
+++ b/packages/cli/src/commands/doctor.ts
@@ -1,13 +1,10 @@
 import { createRequire } from "node:module";
-import { spawnSync } from "node:child_process";
 import { existsSync } from "node:fs";
 import { loadWittgensteinConfig } from "@wittgenstein/core";
 import { firstOutputLine, spawnVersionCheck } from "@wittgenstein/process-runner";
 import type { Command } from "commander";
 import { resolveExecutionRoot } from "./shared.js";
 import { runtimeTierReadiness } from "../tiers.js";
-
-const OPTIONAL_DEPENDENCY_CHECK_TIMEOUT_MS = 10_000;
 
 type DoctorCheckStatus = "ok" | "missing" | "skipped";
 

--- a/packages/codec-image/src/decoders/llamagen/README.md
+++ b/packages/codec-image/src/decoders/llamagen/README.md
@@ -16,7 +16,7 @@ shared [`../weights.ts`](../weights.ts) cache + sha256 resolver and the
 ## Provenance
 
 - **Upstream source repo** — `FoundationVision/LlamaGen` ([github](https://github.com/FoundationVision/LlamaGen),
-  Apache-2.0). The `repoId` + `revision` in the manifest point at the
+  MIT). The `repoId` + `revision` in the manifest point at the
   audited snapshot (`81e41139…`) of the VQ tokenizer source code.
 - **Upstream weights** — `vq_ds16_c2i.pt` (~287 MB FP32, ~72M
   parameters, K=16384, embed dim D=8, downsample factor p=16).

--- a/pnpm-workspace.yaml
+++ b/pnpm-workspace.yaml
@@ -1,8 +1,9 @@
 packages:
   - "packages/*"
-  - "apps/*"
-  - "!apps/wittgenstein-kimi"
-  # apps/_vercel-kimi-out/ is gitignored build output from the Kimi deck.
-  # Excluded from the workspace too so locally generated artifacts can't
-  # trip `pnpm install --frozen-lockfile` for maintainers who built it.
-  - "!apps/_vercel-kimi-out"
+  # Explicit app allowlist instead of `apps/*`: ad-hoc or build-output dirs
+  # (gitignored `apps/_vercel-kimi-out/`, a stray Finder `apps/site copy/`)
+  # must not get pulled into the workspace and trip
+  # `pnpm install --frozen-lockfile`. `apps/wittgenstein-kimi` is
+  # intentionally omitted (standalone Kimi deck, not a workspace member).
+  - "apps/presentation"
+  - "apps/site"

--- a/scripts/check-npm-publish-tarball.mjs
+++ b/scripts/check-npm-publish-tarball.mjs
@@ -6,7 +6,8 @@
  * Per the delivery doctrine
  * (`docs/research/2026-05-13-delivery-and-componentization.md`), a user
  * running `npm install @wittgenstein/cli` must never pull in `research/`,
- * `bench/`, `examples/`, or large binary artifacts. This guard runs
+ * `benchmarks/`, `examples/`, `python/`, `polyglot-mini/`, or large binary
+ * artifacts. This guard runs
  * `npm pack --dry-run --json` per publishable package and verifies the
  * file list contains none of those.
  *

